Extremem/src/main/java/com/amazon/corretto/benchmark/extremem/RelativeTimeMetrics.java

```diff
@@ -49,6 +49,12 @@
  *    coalescing the contents of neighboring buckets, as necessary.
  */
 class RelativeTimeMetrics extends ExtrememObject {
+  // How many buckets used for histogram report?
+  private final int HistoColumnCount = 64;
```

Comment:
Should HisoColumnCount and PageColumns be command-line options?  Might be a good starter task

Reply:
That's a good idea.